xarray/backends/zarr.py

```diff
@@ -633,6 +635,7 @@ def open_store(
         zarr_format=None,
         use_zarr_fill_value_as_mask=None,
         write_empty: bool | None = None,
+        cache_members: bool = False,
```

Comment:
What do you think about changing the default to `True`? I'd like to know if we think there is a downside (risk, performance, etc.) to defaulting to this behavior?

Reply:
There are some weird tests that use `open_store` and `dump_to_store` that can't handle that default.

For nearly all users, the effective default is the value set in `open_zarr` and `to_zarr` (which is `True`, or should be after that one suggestion is merge)

Reply:
i'm up for it! the main risk is whether anyone is actively using `ZarrStore` instances explicitly, with the assumption that they dynamically track the state of the underlying zarr Group. if nobody is doing that, then there's no risk. ofc I have no way of know if this use case is common, but my impression was that `ZarrStore` is internal enough that users are not expected to be using it directly.